pandas/tests/frame/methods/test_isin.py

```diff
@@ -189,3 +189,18 @@ def test_isin_empty_datetimelike(self):
         tm.assert_frame_equal(result, expected)
         result = df1_td.isin(df3)
         tm.assert_frame_equal(result, expected)
+
+    @pytest.mark.parametrize(
+        "category",
```

Comment:
Rather than calling this argument "category" something like "values" or "other" (for example) seems more appropriate